modules/apps/transfer/types/msgs.go

```diff
@@ -131,6 +131,11 @@ func (msg MsgTransfer) GetCoins() sdk.Coins {
 	return coins
 }
 
+// ShouldBeForwarded determines if the transfer should be forwarded to the next hop.
+func (msg MsgTransfer) ShouldBeForwarded() bool {
```

Comment:
wdyt about `HasForwarding` / `HasForwardingHops`?

Usually would prefer this kind of naming convention with `Has` or `Is` prefix

Reply:
Good point. I'll merge this now to get it out of the way, but we can revisit this later when we know more about the unwind and all the other discussion related to this.